src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/AspireAIChat/.template.config/template.json

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
Note: I'm going with a separate template, instead of this being an "Add Aspire Support" for the existing AI Chat Web App Template. This is because a) there's a different approach to configuration and project structure for Aspire support, b) we likely will have additional options for Aspire that you wouldn't see in a non-Aspire scenario, and c) it's a lot easier to merge 2 down to 1 later than to unravel a lot of #if conditional logic.